Testscripts/Linux/utils.sh

```diff
@@ -3550,7 +3550,7 @@ function install_nvme_cli()
         echo "nvme is not installed\n Installing now..."
         check_package "nvme-cli"
         if [ $? -ne 0 ]; then
-            yum -y groupinstall "Development Tools"
+            yum -y --nogpgcheck install gcc gcc-c++ kernel-devel
```

Comment:
better use install_package instead of yum? haven't traced back if install_nvme_cli is used only on redhat/centos, but install_package should be safer to use?

Reply:
install_package is better.